src/test/java/BaseTest.java

```diff
@@ -1,11 +1,83 @@
 import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Parameters;
 
-public class BaseTest {
+import java.time.Duration;
 
+public class BaseTest {
+    public WebDriver driver = null;
+    public String url = "https://qa.koel.app/";
     @BeforeSuite
     static void setupClass() {
         WebDriverManager.chromedriver().setup();
     }
-}
\ No newline at end of file
+    @BeforeMethod
+    @Parameters({"BaseURL"})
```

Comment:
Hello,
Your homework 19 has been reviewed, you have used `@Parameters` for passing BaseURL from the xml config file to the tests. Great! Keep up the good work!
Best wishes from the HW team (Khanh, khanh@testpro.io)

Test PASSED
![image](https://github.com/Noor-01/A51-QA-Automation-Framework/assets/144403954/99772284-9829-47e5-a49b-1b82a3907c90)